frappe/public/js/frappe/widgets/chart_widget.js

```diff
@@ -490,6 +491,14 @@ export default class ChartWidget extends Widget {
 					shortenYAxisNumbers: 1
 				}
 			};
+
+			if (this.chart_doc.custom_options) {
+				let custom_options = JSON.parse(this.chart_doc.custom_options);
```

Comment:
```suggestion
				let custom_options = JSON.parse(this.chart_doc.custom_options || '{}');
```

Reply:
At line no 495 already I've checked custom_options is present or not then why to put or condition?

Reply:
Oh yeah! Missed it!